examples/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ add_subdirectory(simple_intersection)
 
 add_subdirectory(molecular_dynamics)
 
+if(ARBORX_ENABLE_MPI)
```

Comment:
Need
```cmake
if(ARBORX_ENABLE_MPI)
    find_package(MPI REQUIRED)
endif()
```
at the top of the file (for testing installation)

Reply:
isn't it provided by ArborX when MPI is enabled?

Reply:
Yeah, you are right. It built the test, just the test failed somehow.